identifiers.md

```diff
@@ -0,0 +1,68 @@
+# Vidarr Identifiers
+Rather than using incrementing identifiers, Vidarr identifies workflow
+versions, workflow runs, and output files using SHA-256 hashes of key metadata.
+The assumption is that if two objects have the same hash, they must be
+equivalent. All workflow run matching is done by hash matching.
+
+For all hashes, the hash is computed using a SHA-256 of the data described
+below. All strings are converted to UTF-8 encoded bytes. Strings are not
+permitted to contain the `NUL` (zero) byte. Some hashes contain the IDs of
+other hashes. The hashes are encoded as ASCII strings in lowercase hexadecimal
+representation. All JSON objects have keys in alphabetical order.
+
+## Workflow Versions
+A workflow version hash is present for each version of a workflow installed.
+Even if the same WDL file is installed under two different names, there will be
+two different workflow version hashes. It is computed as follows:
+
+- _name_
+- `NUL`
+- _version_
+- `NUL`
```

Comment:
What is the purpose of these nulls in the hash?

Reply:
Added to the docs.